definition_generator/parse_prometheus/prometheus_processor.py

```diff
@@ -2,12 +2,35 @@
 import yaml
 import logging
 import requests
+import re
 
 from prometheus_client.parser import text_string_to_metric_families
 
 RESERVED_NAMESPACE = ["go", "process", "promhttp"]
 logger = logging.getLogger()
 
+DEFAULT_UNIT = 'count'
```

Comment:
I do not understand what `count` means as unit, does it mean is it unitless?

Because when we use type count of a metric it means it cannot decrease

Reply:
my best definition to this would be a `count` is a numeric value. 
I agree that it make some confusion with the type `counter`

Reply:
I agree with Paolo. Doesn't make much sense and it will lead to confusion and maybe problems in the backend? Maybe it0s best to confirm with Julien what the value should be for a generic "items" unit

Reply:
i'm marking this commet as resolved from the conversation with julian https://newrelic.slack.com/archives/GP2EW7ZMG/p1602679475055700